doc/tutorials/run_kata_containers.rst

```diff
@@ -0,0 +1,47 @@
+.. _run-kata-containers:
+
+How to run Kata containers on Service VM
+########################################
+
+This tutorial describes how to install, configure and run Kata containers on
+ACRN Service VM.
+
+Pre-requisites
+**************
+
+.. _kata prerequisites:
+   https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-kata-containers-with-acrn.md#pre-requisites
+
+#. ACRN :ref:`supported hardware <hardware>` with 8GB or 16GB memory size.
+#. You must have 4 CPUs cores available,
```

Comment:
Change to: For default prebuilt ACRN binary in E2E package, You need have 4 CPUs cores or enable “CPU Hyper-Threading” to have 4 CPU Thread for 2 CPU cores.